mobile/src/test/java/org/jraf/android/cinetoday/common/model/movie/MovieAssert.java

```diff
@@ -0,0 +1,194 @@
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.jraf.android.cinetoday.common.model.movie;
+
+import org.fest.assertions.api.AbstractAssert;
+import org.fest.assertions.api.Assertions;
+import org.fest.util.Sets;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.Collection;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class MovieAssert extends AbstractAssert<MovieAssert, Movie> {
+
+    public MovieAssert(Movie actual) {
+        super(actual, MovieAssert.class);
+    }
+
+    private static final Set<String> REQUIRED_SHOWTIME_FIELDS =
+            Sets.newLinkedHashSet("todayShowtimes");
+    private static final Set<String> OPTIONAL_MOVIE_FIELDS =
+            Sets.newLinkedHashSet("actors", "directors", "releaseDate", "trailerUri");
+    private static final Set<String> REQUIRED_MOVIE_DETAIL_FIELDS =
+            Sets.newLinkedHashSet("id", "originalTitle", "localTitle",
+                    "durationSeconds", "genres", "posterUri", "webUri",
+                    "synopsis");
+    private static final Set<String> REQUIRED_MOVIE_LIST_FIELDS =
+            Sets.newLinkedHashSet("id", "localTitle", "releaseDate", "durationSeconds", "genres", "posterUri", "webUri", "todayShowtimes");
+
+    public static MovieAssert assertThat(Movie actual) {
+        return new MovieAssert(actual);
+    }
+
+    public MovieAssert hasId(String id) {
+        isNotNull();
+        Assertions.assertThat(actual.id).isEqualTo(id);
+        return this;
+    }
+
+    public MovieAssert hasOriginalTitle(String originalTitle) {
+        isNotNull();
+        Assertions.assertThat(actual.originalTitle).isEqualTo(originalTitle);
+        return this;
+    }
+
+    public MovieAssert hasLocalTitle(String localTitle) {
+        isNotNull();
+        Assertions.assertThat(actual.localTitle).isEqualTo(localTitle);
+        return this;
+    }
+
+    public MovieAssert hasDirectors(String directors) {
+        isNotNull();
+        Assertions.assertThat(actual.directors).isEqualTo(directors);
+        return this;
+    }
+
+    public MovieAssert hasActors(String actors) {
+        isNotNull();
+        Assertions.assertThat(actual.actors).isEqualTo(actors);
+        return this;
+    }
+
+    public MovieAssert hasReleaseDate(long releaseDate) {
+        isNotNull();
+
+        Assertions.assertThat(actual.releaseDate).isNotNull().hasTime(releaseDate);
+        return this;
+    }
+
+    public MovieAssert hasDurationSeconds(int durationSeconds) {
+        isNotNull();
+        Assertions.assertThat(actual.durationSeconds).isEqualTo(durationSeconds);
+        return this;
+    }
+
+    public MovieAssert hasPosterUri(String posterUri) {
+        isNotNull();
+        Assertions.assertThat(actual.posterUri).isEqualTo(posterUri);
+        return this;
+    }
+
+    public MovieAssert hasTrailerUri(String trailerUri) {
```

Comment:
Most of these `hasXYZ()` methods aren't used yet.  I was too lazy to add assertions on for all the fields for the five test movies.